samples/auth0/program.py

```diff
@@ -0,0 +1,30 @@
+"""Demonstrates AutoKitteh’s Auth0 integration for managing tasks via the Auth0 API."""
+
+from datetime import datetime, timedelta, UTC
+import os
+
+from autokitteh.auth0 import auth0_client
+
+
+auth0 = auth0_client("auth_conn")
+
+ROLE_ID = os.getenv("ROLE_ID", "")
+
+
+def assign_role(event):
+    user = event.data.body.form.get("user_id")
+    auth0.roles.add_users(ROLE_ID, [user])
+    print(f"Assigned role {ROLE_ID} to user {user}")
+
+
+def weekly_user_growth(event):
```

Comment:
If you don't really need `event`, specify `_`
```suggestion
def weekly_user_growth(_):
```